MathlibTest/EuclideanSpace.lean

```diff
@@ -0,0 +1,31 @@
+import Mathlib.Analysis.InnerProductSpace.PiL2
+
+#guard_expr !₂[] = (WithLp.equiv 2 (∀ _ : Fin 0, _)).symm ![]
+#guard_expr !₂[1, 2, 3] = (WithLp.equiv 2 (∀ _ : Fin 3, ℕ)).symm ![1, 2, 3]
+#guard_expr !₁[1, 2, (3 : ℝ)] = (WithLp.equiv 1 (∀ _ : Fin 3, ℝ)).symm ![1, 2, 3]
+
+section delaborator
+
+/-- info: !₂[1, 2, 3] : WithLp 2 (Fin 3 → ℕ) -/
+#guard_msgs in
+#check !₂[1, 2, 3]
+
+set_option pp.mvars false in
+/-- info: !₀[] : WithLp 0 (Fin 0 → ?_) -/
+#guard_msgs in
+#check !₀[]
+
+section var
+variable {p : ENNReal}
+/-- info: (WithLp.equiv p (Fin 3 → ℕ)).symm ![1, 2, 3] : WithLp p (Fin 3 → ℕ) -/
+#guard_msgs in#check !ₚ[1, 2, 3]
+end var
+
+section tombstoned_var
+-- the point of this test is to
```

Comment:
This comment appears unfinished.

Reply:
... check if the reviewers are paying attention :). I've completed this comment.